src/python/zquantum/qaoa/ansatzes/xz_ansatz.py

```diff
@@ -0,0 +1,122 @@
+from zquantum.core.interfaces.ansatz import Ansatz, ansatz_property
+
+from zquantum.core.circuits import Circuit
+from zquantum.core.evolution import time_evolution
+from zquantum.core.openfermion import change_operator_type
+
```

Comment:
Since both ansatzes are quite similar and coming from the same paper, I think it would make sense to put them both in a single file.
Or perhaps even the same class? I think it's worth discussing.